### PR TITLE
feat: #110, Net Promoter Score Data Table

### PR DESCRIPTION
**Pull request by lubegasimon**
allan

alex

### CODE REVIEW

File: src/views/components/NPSdataTable/index.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+    evaluator: "alex",
+    agentName: "thanos",
+    customerName: "wax",
+    customerTel: 6358376,
+    touchPoint: "person-to-person",
+    reason: "call",
+    waitTime: "5 mins",
+    duration: "10 mins",
+    issueResolved: "No",
+    furtherInformation: "Yes",
```
Comment (epicallan) [inline src/views/components/NPSdataTable/index.stories.tsx]: should be `true or false`

File: src/views/components/NPSdataTable/index.tsx [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+import { Profile } from "../UserProfile";
+
+export interface NPS extends TableData {
+  customerName: string;
+  customerTel: number;
+  touchPoint: string;
+  reason: string;
+  waitTime: string;
+  duration: string;
+  issueResolved: string;
```
Comment (epicallan) [inline src/views/components/NPSdataTable/index.tsx]: should be boolean

File: src/views/components/NPSdataTable/index.tsx [hunk 3]
```diff
@@ -0,0 +1,126 @@
...
+  furtherInformation: string;
```
Comment (epicallan) [inline src/views/components/NPSdataTable/index.tsx]:
```suggestion
  furtherInformationGiven: boolean;
```

File: src/views/components/NPSdataTable/index.tsx [hunk 4]
```diff
@@ -0,0 +1,126 @@
...
+  rating: number;
+  ratingReason: string;
+  crmCaptureCorrect: string;
```
Comment (epicallan) [inline src/views/components/NPSdataTable/index.tsx]:
```suggestion
  crmCaptureCorrect: boolean;
```

**Comment by lubegasimon** [comment]
> You can have individual Cell modifier functions to turn them to Yes or No.

how could this be possible?

File: src/views/components/NPSdataTable/index.tsx (hunk 4 above)
Comment (aleku399) [reply src/views/components/NPSdataTable/index.tsx]: but as data comses as a string of yes or no

File: src/views/components/NPSdataTable/index.tsx (hunk 2 above)
Comment (aleku399) [reply src/views/components/NPSdataTable/index.tsx]: but as data comes as a string of yes or no

File: src/views/components/NPSdataTable/index.tsx (hunk 4 above)
Comment (epicallan) [reply src/views/components/NPSdataTable/index.tsx]: @aleku399  the data come as booleans. True or false.

**Pull request merged**